### DISCUSSION TITLE
Data for certificate is invalid

### DISCUSSION THREAD

**Discussion by kcmv** [rcl-ssl/docs#25] [Q&A]
Hi,

After adding the CER certificate to azure application gateway http settings (backend), i got this error.

"Error: Data for certificate is invalid".

Thanks

**Comment by rcladmin** [comment]
what the type of certificate (Stand alone, Stand alone SAN) ? what is the domain name on the certificate?

**Reply by kcmv** [reply]
I have tried Stand alone SAN and Azure DNS SAN. The domain name is www.eperformaxtesting.com

**Reply by rcladmin** [reply]
can you email me the .cer certificates at support@rclapp.com. I will check the data.

**Reply by kcmv** [reply]
Hi @rcladmin, the application gateway is now working. Turns out that I only need the PFX and not the CER. Thank you.

**Reply by rcladmin** [reply]
OK
